config/application.php

```diff
@@ -52,10 +52,18 @@
 /**
  * DB settings
  */
-Config::define('DB_NAME', env('DB_NAME'));
-Config::define('DB_USER', env('DB_USER'));
-Config::define('DB_PASSWORD', env('DB_PASSWORD'));
-Config::define('DB_HOST', env('DB_HOST') ?: 'localhost');
+$hasDsn = env('DATABASE_URL') !== null ? true : false;
```

Comment:
This could be reduced to:
```
$hasDsn = env('DATABASE_URL') !== null;
```
But the line isn't necessary, so I'd suggest removing it. There's no need to use the `$hasDsn` variable.